charters/UploadService/charter.md

```diff
@@ -0,0 +1,78 @@
+# Upload Service
+
+## Description
+The Upload Service enables fast, cost-effective upload, staging, and validation of data submitted to the Human Cell Atlas (HCA) Data Coordination Platform (DCP) from across the world.
+
+## Objectives
+1. Enable submitters to upload files quickly and cost-effectively to temporary *staging areas* for subsequent submission to the Data Storage Service (DSS)
```

Comment:
As above. I think I'd prefer *upload areas* rather than *staging areas*, this terminology was changed in the past to avoid confusion with the staging environment